quran/quran.com-frontend-next
# Change title of pages dynamically

### Summary
This PR adds title to pages dynamically.

### Test Plan
1. Go to Surah Al Fatihah and the title of the page should be `Surah Al-Fatihah - 1-7 - Quran.com`.
2. Go to the first verse of Surah al fatihah's page and the title of the page should be `Surah Al-Fatihah - 1 - Quran.com`

### Screenshots

| Before | After |
| ------ | ------ |
| <img width="244" alt="Screen Shot 2021-08-07 at 12 13 16" src="https://user-images.githubusercontent.com/15169499/128588980-402d6737-c180-4a15-8f30-36b079a36205.png"> | <img width="242" alt="Screen Shot 2021-08-07 at 12 13 11" src="https://user-images.githubusercontent.com/15169499/128588979-628c3168-c1ec-40c4-826a-da52307415ad.png"> |
| | <img width="244" alt="Screen Shot 2021-08-07 at 12 13 42" src="https://user-images.githubusercontent.com/15169499/128588981-d3a59669-af14-4005-a8c1-86ef3b8b38ce.png"> |
